src/psyclone/psyir/nodes/psy_data_node.py

```diff
@@ -137,8 +137,7 @@ def __init__(self, ast=None, children=None, parent=None, options=None):
         # name of the PSyData derived type) but we keep a list for future
         # extensibility.
         _PSyDataSymbol = namedtuple("_PSyDataSymbol", "name symbol_type")
-        self.imported_symbols = [_PSyDataSymbol(self.type_name,
-                                                DataTypeSymbol)]
+        self.import_symbols = [_PSyDataSymbol(self.type_name, DataTypeSymbol)]
```

Comment:
A small point but I prefer the original `imported_symbols` - any reason for the change?

Reply:
I agree `imported_symbols` feels more natural. Changed.